unique-universes/src/views/shoot.py

```diff
@@ -0,0 +1,295 @@
+from __future__ import annotations
+
+import dataclasses
+import math
+import random
+from asyncio import sleep
+from typing import TYPE_CHECKING, cast
+
+import disnake
+
+# weird import but it's to avoid circular imports
+import src.exts.minigames as games
+from src.bot import Universe
+from src.database import PlayerExistsError
+
+if TYPE_CHECKING:
+    from src.database import PlayerData
+
+
+# acceleration directed to the center of Earth
+# measured in m/s^2
+EARTH_ACCELERATION = 9.81
```

Comment:
Speaking of too much information, could account for the gravity in various points on Earth ;)